frontend/widgets/SwellWelcome/components/LoggedOutView/style.js

```diff
@@ -0,0 +1,70 @@
+import { css } from 'glamor';
+
+/**
+ * @param {string} color font color for widget
+ * @param {string} backgroundImage background image for widget
+ * @returns {Function}
+ */
+const container = (color, backgroundImage) => css({
+  color,
+  backgroundImage,
```

Comment:
```suggestion
  background,
```
in settings:
"loggedOutBackgroundImage" --> loggedOutBackground

with this change it will be possible to use simple bg color too

Reply:
we should also add background-size cover in the default value for "loggedOutBackground"

Reply:
Updated. Thanks for the suggestion.